src/cipher/encryptingstorage/__init__.py

```diff
@@ -172,6 +199,58 @@ def decrypt(data):
     data = decompress(data)
     return data
 
+
+def encrypt_file(filename):
+    """ Reads the file "filename" and overwrites it
+    with its data encrypted.
+
+    :param filename: File to encrypt and override.
+    """
+
+    tmp_file = filename + '.enc'
+    with open(filename, 'rb') as fsrc:
+        with open(tmp_file, 'wb') as fdst:
+            encrypt_util.ENCRYPTION_UTILITY.encrypt_file(fsrc, fdst)
+
+    os.remove(filename)
+    os.rename(tmp_file, filename)
+
+
+def decrypt_file(filename, blob_dir):
+    """ Reads the import "filename" decrypts it
+    and writes the decrypted data to a temp file in
+    tmp directory parallel to the 'blobstorage'.
+
+    IF the file doesn't exists in temp_dir
+    ELSE it just returns the path in temp_dir.
+
+    :param filename: Encrypted file to read.
+    :param blob_dir: Path to the blob storage.
+
+    :returns:   The path to the temporary file.
+
+    TODO: Currently theres no code that handles the deletion of the file.
```

Comment:
Yes, thats a big thing. But currently theres `filename` passing instead of `file descriptor` passing in the Blob code, which means we currently don't know when the file gets closed.

I'm sure i'll get the time to implement monkey patches of ZODB for that and maybe upstream patches.